shared/k8s/debug.go

```diff
@@ -0,0 +1,22 @@
+package k8s
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func FetchValidatorPrivateKeys(clientset *kubernetes.Clientset) (map[string]string, error) {
+	var validatorKeysMap = make(map[string]string)
+
+	private_keys_secret, err := clientset.CoreV1().Secrets("default").Get(context.TODO(), "v1-localnet-validators-private-keys", metav1.GetOptions{})
+	if err != nil {
+		panic(err)
+	}
+
+	for id, privHexString := range private_keys_secret.Data {
+		validatorKeysMap[id] = string(privHexString)
```

Comment:
Is the hex string stored as utf8 encoded bytes in `Data`? If so, just add a comment

Reply:
The documentation says it's base64 encoded but it's not. simply casting to string returns the actual encoded private key. 

![image](https://user-images.githubusercontent.com/29378614/219231211-e4181bbb-5875-478b-b0c5-44d61e19fcfd.png)

An experiment I just did:

![image](https://user-images.githubusercontent.com/29378614/219231259-d5079f63-3064-41dd-9f29-9a86fc6cbb0d.png)

pk represents the base64 decoded value from what is not base64 encoded

Reply:
So we're encoding it to base64, but then casting it to a byte slice which is just "a weird thing to do" unless you want to get the byte count of the string or something.

Sound correct? Can you add a TODO/BUG next to the `Data` attribute so we don't lose this.

Reply:
No that's K8S documentation, they return a []byte and say that it's base64 encoded but it's not.